drizzlepac/pars/hap_pars/default_parameters/acs/wfc/acs_wfc_alignment_all.json

```diff
@@ -17,13 +17,13 @@
       },
     "generate_source_catalogs":
       {
-        "box_size":50,
+        "box_size":27,
         "win_size":3,
         "nsigma":5.0,
         "centering_mode": "starfind",
-        "bkg_estimator": "MedianBackground",
+        "bkg_estimator": "SExtractorBackground",
         "rms_estimator": "StdBackgroundRMS",
```

Comment:
This is the only configuration file which has the bkg_estimator set to SExtractorBackground.  I just want to make sure this is intended.

Reply:
Actually, that is an oversight, as WFC3/UVIS should probably also use the same estimator.  This will be addressed shortly in a patch commit.